packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/actionTypes.js

```diff
@@ -0,0 +1,16 @@
+export const SET_ONFIDO_ENABLED = '@KYC.SET_ONFIDO_ENABLED'
+export const SET_VERIIFICATION_STEP = '@KYC.SET_VERIIFICATION_STEP'
+export const SET_PERSONAL_STEP = '@KYC.SET_PERSONAL_STEP'
+export const SET_EMAIL_STEP = '@KYC.SET_VERIIFICATION_EMAIL_STEP'
+export const SET_SMS_STEP = '@KYC.SET_VERIIFICATION_SMS_STEP'
+
+export const UPDATE_PERSONAL_STEP = '@EVENT.KYC_UPDATE_PERSONAL_STEP'
+export const EDIT_EMAIL = '@EVENT.KYC_EDIT_EMAIL'
+export const EDIT_SMS = '@EVENT.KYC_EDIT_SMS'
+
+export const UPDATE_EMAIL = '@EVENT.KYC_UPDATE_EMAIL'
+export const VERIFY_EMAIL = '@EVENT.KYC_VERIFY_EMAIL'
```

Comment:
UI needs know when this is happening, the button should go to `disabled` state and show HeartbeatLoader.

Reply:
ok, will update it